pom.xml

```diff
@@ -56,6 +56,10 @@
       <resource>
         <directory>plugins/credentials-plugin/src/main/resources</directory>
       </resource>
+      <resource>
+        <directory>plugins/credentials-plugin/src/main/webapp</directory>
+        <targetPath>webapp</targetPath>
```

Comment:
So the interesting bit is, as currently proposed, plugin `webapp/` resources would like in a subdirectory, while core `webapp/` resources would not. This is due to `#getPluginResource` being exclusively used for plugin webapp resources, while `#getResource` is used for other resources as well.